array-methods/filter.js

```diff
@@ -0,0 +1,23 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evenNumbers = numbers.filter((x) => {
+  return x % 2 === 0;
+});
```

Comment:
You can use the shorthand arrow notation to make this even more succinct:
`const evenNumbers = numbers.filter((x) => x % 2 === 0);`